test/email.routes.test.js

```diff
@@ -127,6 +129,20 @@ describe("email.routes.test", () => {
       });
   });
 
+  it("rejects emails sent to unknown mailing list", (done) => {
+
+    const unknownMailingListWebhook = _.defaults({ recipient: 'unknown@testcollective.opencollective.com' }, webhookBody);
```

Comment:
actually I guess it's the same to use `Object.assign` (no need for external lib)?